frontend/src/pages/notebookController/screens/server/BrowserTabPreferenceCheckbox.tsx

```diff
@@ -16,7 +16,7 @@ const BrowserTabPreferenceCheckbox: React.FC = () => {
         name="checkbox-notebook-browser-tab-preference"
         isChecked={isUsingCurrentTab}
         label="Start server in current tab"
-        onChange={handleChange}
+        onChange={(e, checked: boolean) => handleChange(checked)}
```

Comment:
FWIW, TS infers types from props
```suggestion
        onChange={(e, checked) => handleChange(checked)}
```
<img width="684" alt="image" src="https://github.com/opendatahub-io/odh-dashboard/assets/8126518/54af0f70-7342-4fc5-9743-a70b55055555">